### ISSUE TITLE
Conda-env should not be installable outside of the root miniconda/anaconda installation

### ISSUE THREAD

**Issue by msarahan** [conda/conda-env#179]
Otherwise, Activate.bat on Windows behaves very poorly, and you lose the ability to deactivate back to root environment

**Comment by malev** [comment]
as far as I know, `conda-env` is a hard dependency of `conda` therefore you must have conda-env installed in the root environment as soon as you finish installing conda. After that you don't need to install conda-env again and you shouldn't install it in an environment other than the root env.
Maybe we can add a script that verifies where are you installing conda-env and shows you an error if you are not using the root environment.

**Comment by msarahan** [comment]
yes, it shouldn't need to be installed, but it is getting picked up automatically in my development (see https://github.com/conda/conda/issues/1705).  This is causing a lot of confusion.

**Comment by malev** [comment]
@msarahan you are taking about a PR for conda-env. which one is that?

**Comment by msarahan** [comment]
https://github.com/conda/conda-env/pull/178

**Comment by ijstokes** [comment]
I'd like to make a proposal that `conda-env` and `anaconda-client` need to be merged into the `conda` package and be one and the same tool.  Besides the fact that I think it is veru confusing to have `conda` and `anaconda` be two different CLI commands, we have the issues mentioned here, as well as:

```bash
BUILD END: opencv-2.4.11-py27_0
Nothing to test for: opencv-2.4.11-py27_0
# If you want to upload this package to anaconda.org later, type:
#
# $ anaconda upload /Users/ijstokes/anaconda/conda-bld/osx-64/opencv-2.4.11-py27_0.tar.bz2
#
# To have conda build upload to anaconda.org automatically, use
# $ conda config --set anaconda_upload yes

(numba_cuda)ijstokes@petawawa conda-recipes (master) $ anaconda upload /Users/ijstokes/anaconda/conda-bld/osx-64/opencv-2.4.11-
py27_0.tar.bz2
-bash: anaconda: command not found
```
So I think this is a weird situation.  There are a set of "meta" commands that all should only live in the root environment.  They can be "logically distinct", but I believe i) they should all be part of the `conda` package; and ii) they should all be in the `root` environment (and **only** in the `root` environment).  They are:

`conda-env`
`conda-build`
`anaconda-client`

For consideration by @ilanschnell @msarahan @malev @srossross and @csoja

**Comment by nicoddemus** [comment]
I personally agree, specially in the case of `conda` and `conda-env`, although I agree that it also appears to make sense for `conda-build`.

**Comment by Korijn** [comment]
:+1: @ijstokes

**Comment by ilanschnell** [comment]
I disagree.  Almost 2 years ago conda-build was specifically moved out of
conda to make conda smaller and easier to maintain.  The ability to install
packages into the root environment only can usually be achieved by making
them depend on conda.  However, in the case of conda-env this is not so
easy, as conda depends on conda-env.

**Comment by teoliphant** [comment]
anaconda-client can also *not* be merged into conda as it deals with separate things.    Some of the features of anaconda-client could be refactored into conda as needed.   But the anaconda command is really about interacting with Anaconda Cloud (anaconda.org) and is different.
